src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -398,92 +412,97 @@ else if (loansParametersMatch(req, Integer.MAX_VALUE)) {
         req.response()
           .setStatusCode(200)
           .putHeader("content-type", "application/json")
-          .end(readMockFile(mockDataFolder + "/item_book1.json"));
+          .end(readMockFile(mockDataFolder + "/items/item_book1.json"));
       } else if (req.path().equals("/inventory/items/" + itemBook2Id)) {
         req.response()
           .setStatusCode(200)
           .putHeader("content-type", "application/json")
-          .end(readMockFile(mockDataFolder + "/item_book2.json"));
+          .end(readMockFile(mockDataFolder + "/items/item_book2.json"));
       } else if (req.path().equals("/inventory/items/" + itemBook3Id)) {
         req.response()
           .setStatusCode(200)
           .putHeader("content-type", "application/json")
-          .end(readMockFile(mockDataFolder + "/item_book3.json"));
+          .end(readMockFile(mockDataFolder + "/items/item_book3.json"));
       } else if (req.path().equals("/inventory/items/" + goodItemId)) {
         req.response()
           .setStatusCode(200)
           .putHeader("content-type", "application/json")
-          .end(readMockFile(mockDataFolder + "/item_book4.json"));
+          .end(readMockFile(mockDataFolder + "/items/item_book4.json"));
       } else if (req.path().equals("/inventory/items/" + checkedoutItemId)) {
         req.response()
           .setStatusCode(200)
           .putHeader("content-type", "application/json")
-          .end(readMockFile(mockDataFolder + "/item_checkedout.json"));
+          .end(readMockFile(mockDataFolder + "/items/item_checkedout.json"));
       }  else if (req.path().equals("/inventory/items/" + intransitItemId)) {
         req.response()
           .setStatusCode(200)
           .putHeader("content-type", "application/json")
-          .end(readMockFile(mockDataFolder + "/item_intransit.json"));
+          .end(readMockFile(mockDataFolder + "/items/item_intransit.json"));
+      }  else if (req.path().equals("/inventory/items/" + missingHoldingsRecordId)) {
+        req.response()
+          .setStatusCode(200)
```

Comment:
Check why 200

Reply:
if I change status to 400 either 422 the vert.x just is hanging

Reply:
Sorry, I've completely confused. We return 200 because there is no error. We just mock a request for inventory items so that it returns an item to us without a field holdingsRecordId